vspace/src/main/webapp/WEB-INF/views/staff/spaces/space.jsp

```diff
@@ -34,7 +34,6 @@ $( document ).ready(function() {
 	$("#addSpaceLinkButton").click(function(e) {
 		$("#bgImage").on("click", function(e){
 		    e.preventDefault();
-		    $("#arrow").remove();
```

Comment:
doesn't this need to be here? Isn't that the code that makes sure there is no second arrow when clicking on the image a second time?